webrtc/dtls/dtls_transport.nim

```diff
@@ -89,23 +89,27 @@ proc accept*(self: Dtls): Future[DtlsConn] {.async: (raises: [CancelledError, We
   res.acceptInit(self.ctr_drbg, self.serverPrivKey, self.serverCert, self.localCert)
 
   while true:
-    try:
-      self.connections[res.raddr] = res
-      await res.dtlsHandshake(true)
-      asyncSpawn self.cleanupDtlsConn(res)
-      break
-    except WebRtcError as exc:
-      trace "Handshake fails, try accept another connection",
-            remoteAddress = res.raddr, error = exc.msg
-      self.connections.del(res.raddr)
-      res.conn = await self.transport.accept()
+    if res.raddr.family == AddressFamily.IPv4 or res.raddr.family == AddressFamily.IPv6:
```

Comment:
why would a connection be created with an address that doesn't satisfy that?

Reply:
Done.